src/stores/Inventory.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+import { observable, action, makeObservable, computed } from 'mobx';
+import { Item } from './Item';
+
+export class Inventory {
+    constructor() {
+        this.item = [];
+        this.length = 0;
```

Comment:
I don't think you need this. The computed property of `numItems` gives you the length.